src/core/pluginmanager.cpp

```diff
@@ -455,3 +457,27 @@ QString PluginManager::findProjectPlugin( const QString &projectPath )
   }
   return QString();
 }
+
+void PluginManager::callPluginMethod( const QString &uuid, const QString &methodName )
+{
+  if ( !mAvailableAppPlugins.contains( uuid ) )
+  {
+    return;
+  }
+
+  const QString pluginPath = mAvailableAppPlugins[uuid].path();
+  if ( !mLoadedPlugins.contains( pluginPath ) )
+  {
+    return;
+  }
+
+  const QPointer<QObject> object = mLoadedPlugins[pluginPath];
+
+  const char *normalizedSignature = QMetaObject::normalizedSignature( ( methodName + "()" ).toStdString().c_str() );
+  const int methodExists = object->metaObject()->indexOfSlot( normalizedSignature );
```

Comment:
methodIndex would be a better name. Or have methodExists be a bool and add >= 0 at the end of this line.